.github/workflows/deprecations.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Crawl Deprecations
+
+on:
+  push:
+    branches:
+      - setup-git-action
```

Comment:
Should this be removed?

Reply:
yeah! is was just for testing. It should be the main branch